### DISCUSSION TITLE
Language change?

### DISCUSSION THREAD

**Discussion by EmotionChild** [EllieBotDevs/Ellie-bot#114] [Q&A] [answered]
So I have been thinking of changing the language that this project runs on and I am thinking of using Java but I want your thoughts.

**Comment by EmotionChild** [comment]
Small addition if you want to help us with this let me know over Discord or over email (my email is in my bio)

**Reply by AidanTheBandit** [reply]
I've had good experiences with my bots on Java although JS is supposed to be better I've heard

**Reply by EmotionChild** [reply]
That is some good information the main reason for this is cause I am getting quite burnt out of using JS and I am looking for something new so I can make things that are more stable

**Reply by AidanTheBandit** [reply]
I'll be able to contribute more if you do choose Java since that's the language I use the most

**Reply by EmotionChild** [reply]
I am really considering switching to java since I have not had any issues with it at all

**Reply by EmotionChild** [reply]
Started in [#116](https://github.com/EllieBotDevs/Ellie-bot/pull/116)

**Comment by EmotionChild** [comment, marked as answer]
Locked as this has been started
